testplan/runnable/interactive/base.py

```diff
@@ -736,30 +737,37 @@ def reload(self, rebuild_dependencies=False):
     def reload_report(self):
         """Update report with added/removed testcases"""
         new_report = self._initial_report()
-        for multitest in self.report.entries:  # multitest level
-            for suite_index, suite in enumerate(multitest.entries):
-                new_suite = new_report[multitest.uid][suite.uid]
-                for case_index, case in enumerate(suite.entries):
-                    try:
-                        if isinstance(case, TestGroupReport):
-                            for param_index, param_case in enumerate(
-                                case.entries
-                            ):
-                                try:
-                                    new_report[multitest.uid][suite.uid][
-                                        case.uid
-                                    ].entries[param_index] = case[
-                                        param_case.uid
-                                    ]
-                                except (KeyError, IndexError):
-                                    continue
-                        else:
-                            new_report[multitest.uid][suite.uid].entries[
-                                case_index
-                            ] = suite[case.uid]
```

Comment:
I think this is where the original problem is right? that we assume case of a particular uid is still at the same index?

Reply:
Can we just fix this bit?